flocker/node/agents/test/test_blockdevice.py

```diff
@@ -2281,6 +2281,45 @@ def setUp(self):
     return Tests
 
 
+class IProfiledBlockDeviceAPITestsMixin(object):
+    """
+    Tests to perform on ``IProfiledBlockDeviceAPI`` providers.
+    """
+    def test_interface(self):
+        """
+        The API object provides ``IProfiledBlockDeviceAPI``.
+        """
+        self.assertTrue(
+            verifyObject(IProfiledBlockDeviceAPI, self.api)
+        )
+
+    def test_profile_respected(self):
+        """
+        Verify that the profile for a volume created with one of the mandatory
+        profiles is respected.
+        """
+        for profile in [u'gold', u'silver', u'bronze']:
```

Comment:
I suggest default belongs outside the class, but yeah having explicit constants is a good idea.